### DISCUSSION TITLE
Build android not working on macOS

### DISCUSSION THREAD

**Discussion by NgVThangBz** [axmolengine/axmol#1351] [Q&A] [answered]
im use:
`axmol new -p com.bazeni.BZSketch -d /Users/baezenisoft/Documents/GitHub/BzSketch-Axmol  -l cpp --portrait testAndroid`
`cd testAndroid`
`axmol build -p android -a arm64`
and when i open project with android studio,
im receive error:

![Screenshot 2023-10-05 at 17 17 44](https://github.com/axmolengine/axmol/assets/125545428/3099064c-e9ee-4ae6-840c-3727687e2074)
Cause: com/android/tools/lin/model/LintModelSeverity has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
i need update anything?

**Comment by halx99** [comment]
Check does AX_ROOT of launchctl is correct, if not, rerun `pwsh ./setup.ps1` will auto set it: `https://github.com/axmolengine/axmol/blob/dev/setup.ps1#L230` 

```sh
launchctl getenv AX_ROOT
```

**Reply by halx99** [reply, marked as answer]
android studio 2022 recommended
